docs/production/authentication-methods.md

```diff
@@ -697,11 +697,24 @@ enabling `zproject.backends.GenericOpenIdConnectBackend` in
 `AUTHENTICATION_BACKENDS` and following the steps outlined in the
 comment documentation in `/etc/zulip/settings.py`.
 
-Note that `SOCIAL_AUTH_OIDC_ENABLED_IDPS` only supports a single backend
+Note that `SOCIAL_AUTH_OIDC_ENABLED_IDPS` only supports a single IdP currently.
 
 The Return URL to authorize with the provider is
 `https://yourzulipdomain.example.com/complete/oidc/`.
 
+If you'd like first-time users who authenticate via the configured IdP
+to have their account seemlessly created instead of being asked to confirm
```

Comment:
`to have their account seemlessly created` not sure if that's the best phrasing, given that without `SOCIAL_AUTH_OIDC_FULL_NAME_VALIDATED` they'll go through the regular registration form - but talking more precisely  about how the "confirm registration" page  will be skipped, but the registration form will appear may be a bit too much of drowning the reader in low-level details?

Reply:
I tried to simplify this a bit, but I think it's worth explaining clearly since these details can be important.